findoutlie/metrics.py

```diff
@@ -21,12 +20,8 @@ def dvars(img):
         One-dimensional array with n-1 elements, where n is the number of
         volumes in `img`.
     """
-    # Hint: remember 'axis='.  For example:
-    # In [2]: arr = np.array([[2, 3, 4], [5, 6, 7]])
-    # In [3]: np.mean(arr, axis=1)
-    # Out[2]: array([3., 6.])
-    #
-    # You may be be able to solve this in four lines, without a loop.
-    # But solve it any way you can.
-    # This is a placeholder, replace it to write your solution.
-    raise NotImplementedError('Code up this function')
+    data = img.get_fdata()
+    n_voxels = data.shape[0] * data.shape[1] * data.shape[2]
+    data_2d = data.reshape(n_voxels, data.shape[-1])
```

Comment:
Numpy will actually calculate this for you...

```suggestion
    data_2d = data.reshape(-1, data.shape[-1])
```

Reply:
Thanks for the hint!